created Player.py, FallingObject.py modules, renamed the main file in…

freddycapo opened the pull request:
…to "game.py" from "play.py"

freddycapo committed:
created Player.py, FallingObject.py modules, renamed the main file into "game.py" from "play.py"

freddycapo committed:
add Player.py and FallingObject.py, changed the name of the main file into game.py

freddycapo committed:
Merge branch 'code_restructuring' of https://github.com/freddycapo/covidgame into code_restructuring

manamana88 commented on src/game.py:
```diff
@@ -19,85 +21,17 @@
 
 class Mask(FallingObject):
-    def __init__(self, x, y, width, heigth,vel,sprite):
-        super().__init__(x, y, width, heigth)
+    def __init__(self, x, y, width, heigth,screen_heigth,vel,sprite):
+        super().__init__(x, y, width, heigth,screen_heigth)
         self.vel=vel
         self.sprite=sprite
     pass
```
Hi Fede,
move Mask class in a separate file as well

manamana88 commented on src/game.py:
```diff
@@ -19,85 +21,17 @@
 
 class Virus(FallingObject):
-    def __init__(self, x, y, width, heigth,vel,sprite):
-        super().__init__(x, y, width, heigth)
+    def __init__(self, x, y, width, heigth,screen_heigth,vel,sprite):
+        super().__init__(x, y, width, heigth,screen_heigth)
         self.vel=vel
         self.sprite=sprite
     pass
```
Hi Fede,
move Virus class in a separate file as well

manamana88 reviewed (changes requested):
Hi Fede,
I like the branch and the pull request! You are starting to use GitHub like a pro!!

Remove the two cache objects and add the folder **src/__pycache__/** to **.gitignore**
Cache files should not be  part of the code base because they are just temporary files that are needed locally

freddycapo committed:
added Mask.py and Virus.py, updated .gitignore

freddycapo committed:
create requirements.txt

manamana88 committed:
Improved workflow to support SDL library

manamana88 committed:
Test

This is a test to verify if the pullrequest check is using the master workflow file or the branch one

freddycapo committed:
changed setup.py

manamana88 committed:
Included dependencies in build

manamana88 committed:
Removed tests
